src/components/ArticleStats/ArticleStats.tsx

```diff
@@ -40,8 +35,9 @@ export const ArticleStats = ({ readTime, date }: ArticleStatsProps) => {
   }
 
   const getDate = (articleDate: string) => {
-    const formatMethod = 'PP'
-    const date = format(new Date(articleDate), formatMethod)
+    dayjs.extend(utc)
```

Comment:
Melhorar composicao de componentes (ArticleStats, ArticleDate, ArticleReadime, etc),

Reply:
Check this [commit](https://github.com/joaooliveiradev/conduit-frontend/pull/43/commits/b4a84412f7e418e76124cc2f7e7a5938c4c42a77).